agent/dns_test.go

```diff
@@ -3556,40 +3556,48 @@ func TestDNS_ServiceLookup_TagPeriod(t *testing.T) {
 		t.Fatalf("Bad: %#v", in)
 	}
 
-	m := new(dns.Msg)
-	m.SetQuestion("v1.primary.db.service.consul.", dns.TypeSRV)
-
-	c := new(dns.Client)
-	in, _, err = c.Exchange(m, a.DNSAddr())
-	if err != nil {
-		t.Fatalf("err: %v", err)
+	validQuestions := []string{
```

Comment:
Modified this test to use an array of questions, including the previously-existing standard lookup and the new RFC 2782 lookup format for tag-based filtering